.travis.yml

```diff
@@ -8,4 +8,6 @@ matrix:
 script:
   - cargo build
   - cargo test --all --exclude uint
-  - cd uint/ && cargo test --features=std,impl_quickcheck_arbitrary
+  - cd uint/ && cargo test --features=std,impl_quickcheck_arbitrary && cd ..
+  - cd hashdb/ && cargo test --no-default-features && cd ..
```

Comment:
Do we need to pass these crates to `--exclude` above, or are we going to have some `std/no-std`-specific tests in the future?

Reply:
At the moment `hashdb` only defines traits and have no tests, so `cargo test` amounts to essentially typechecking. `plain_hasher` has one test so this runs it both ways (std/no_std). 
I think this is ok for now.